Tests/configure_and_test_integration_instances.py

```diff
@@ -0,0 +1,472 @@
+    ]
+    modded_integration_files = [
+        file_path for file_path in modified_files if
+        isinstance(file_path, str) and checked_type(file_path, all_integration_regexes)
+    ]
+
+    new_integrations_names = []
+    for integration_file_path in added_integration_files:
+        integration_yaml = get_yaml(integration_file_path)
+        integration_name = integration_yaml.get('name')
+        if integration_name:
+            new_integrations_names.append(integration_name)
+    modded_integrations_names = []
+    for integration_file_path in modded_integration_files:
+        integration_yaml = get_yaml(integration_file_path)
+        integration_name = integration_yaml.get('name')
+        if integration_name:
+            modded_integrations_names.append(integration_name)
+    return new_integrations_names, modded_integrations_names
+
+
+def is_content_updating(server, username, password):
+    '''Configure Demisto Client and make request to check if content is updating'''
+    # Configure Demisto Client
+    c = demisto_client.configure(base_url=server, username=username, password=password, verify_ssl=False)
+
+    msg = '\nMaking "Get" request to server - "{}" to check if content is installing.'.format(server)
+    print(msg)
+
+    # make request to check if content is updating
+    response_data, status_code, _ = demisto_client.generic_request_func(self=c, path='/content/updating',
+                                                                        method='GET', accept='application/json')
+
+    if status_code >= 300 or status_code < 200:
+        result_object = ast.literal_eval(response_data)
+        message = result_object.get('message', '')
+        msg = "Failed to check if content is installing - with status code " + str(status_code) + '\n' + message
+        print_error(msg)
+        return 'request unsuccessful'
+    else:
+        return response_data
+
+
+def get_content_installation(client):
+    '''Make request for details about the content installed on the demisto instance'''
+    host = client.api_client.configuration.host
+    msg = '\nMaking "POST" request to server - "{}" to check installed content.'.format(host)
+    print(msg)
+
+    # make request to installed content details
+    response_data, status_code, _ = demisto_client.generic_request_func(self=client, path='/content/installed',
+                                                                        method='POST')
+
+    result_object = ast.literal_eval(response_data)
+    if status_code >= 300 or status_code < 200:
+        message = result_object.get('message', '')
+        msg = "Failed to check if installed content details - with status code " + str(status_code) + '\n' + message
+        print_error(msg)
+    return result_object.get('release', ''), result_object.get('assetId', 0)
+
+
+def set_integration_params(integrations, secret_params, instance_names):
```

Comment:
don't we have something of that sort in test_integration ?

Reply:
yes but I got rid of unnecessary arguments and removed some lines